src/components/HintPage.jsx

```diff
@@ -37,6 +41,26 @@ const DumbHintPage = props => {
     }
   }
 
+  useEffect(() => {
+    const checkExistingHint = async () => {
+      try {
+        await client.getStackClient().fetchJSON('GET', '/settings/hint')
```

Comment:
You have a `SettingsCollection.get('hint')` to do that :p

Reply:
No need for a new permission to request this route?

Reply:
> You have a `SettingsCollection.get('hint')` to do that :p
I didn't know ! Thanks

> No need for a new permission to request this route?

Just checked, we actually had only PUT permission in the manifest, but the call on GET /settings/hint works with just that :thinking: